webpack/assets/javascripts/foreman_hosts.js

```diff
@@ -0,0 +1,26 @@
+export let pluginEditAttributes = {
+  architecture: [],
+  os: [],
+  medium: [],
+  image: []
+};
+
+export function getAttributesToPost(selectionType) {
+  const defaultAttributes = {
+    'architecture': ['architecture_id', 'organization_id', 'location_id'],
+    'os': ['operatingsystem_id', 'organization_id', 'location_id'],
+    'medium': ['medium_id', 'operatingsystem_id', 'architecture_id'],
+    'image': ['medium_id', 'operatingsystem_id', 'architecture_id', 'model_id']
+  };
+  let attrsToPost = defaultAttributes[selectionType];
+
+  if (attrsToPost === undefined) {
+    return [];
+  }
+
+  if (window.tfm.hosts.pluginEditAttributes[selectionType]) {
```

Comment:
I think there's no need to fully qualify the `pluginEditAttributes` object here?